app/src/controllers/authenticationController.js

```diff
@@ -51,4 +55,24 @@ export default class AuthenticationController {
       }
     );
   }
+
+  /**
+   * The metadata for this Service Provider.
+   *
+   * @param req
+   * @param res
+   */
+  metadata(req: express$Request, res: express$Response) {
```

Comment:
accessing global vars makes testing hard, I suggest to refactor this in two steps:

``` js
function getMetadataHandler(certPath, spidStrategy) {
  return (req, res) => {...};
}

// ...

{
  // ...
  metadata: getMetadataHandler(process.env.SAML_CERT_PATH || "./certs/cert.pem", spidStrategy)
}
```